hsph.js

```diff
@@ -0,0 +1,73 @@
+(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+})(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+
+ga('create', 'UA-64317003-1', 'auto');
+ga('require', 'displayfeatures');
+ga('require', 'linkid', 'linkid.js');
+ga('send', 'pageview');
+
+//Load jQuery library using plain JavaScript
+(function(){
+	var kscript = document.createElement('script');
+	kscript.type = 'text/javascript';
+	kscript.src = 'https://cdnapisec.kaltura.com/p/1446471/sp/144647100/embedIframeJs/uiconf_id/30101351/partner_id/1446471';
+	document.getElementsByTagName('head')[0].appendChild(kscript);
+  
+	var newscript = document.createElement('script');
+	newscript.type = 'text/javascript';
+	newscript.src = 'https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js';
```

Comment:
jQuery should already be present in Canvas pages; is it necessary to load it again?